Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -947,17 +947,26 @@ protected IEnumerable<DataGridColumn<TItem>> DisplayableColumns
         /// <summary>
         /// Returns true if EmptyTemplate is set and Data is null or empty.
         /// </summary>
-        protected bool IsEmptyTemplateVisible => !IsLoadingTemplateVisible && !IsNewItemInGrid && EmptyTemplate != null && ( Data == null || !Data.Any() );
+        protected bool IsEmptyTemplateVisible
+            => !IsLoadingTemplateVisible && !IsNewItemInGrid && EmptyTemplate != null && ( Data == null || !Data.Any() );

```

Comment:
Might be worth creating an internal extension for collections, EmptyOrNull and/or AnyNotNull

Reply:
That is good idea.

Reply:
Added IsNullOrEmpty extension

Reply:
Ya looks better.